api/types/types_drivers_executor.go

```diff
@@ -159,7 +182,8 @@ type ProvidesStorageExecutorCLI interface {
 // StorageExecutorCLI provides a way to interact with the CLI tool built with
 // the driver implementations of the StorageExecutor interface.
 type StorageExecutorCLI interface {
-	StorageExecutorWithSupported
+	StorageExecutorFunctions
+	StorageExecutorWithMount
```

Comment:
Looks like the `StorageExecutorWithSupported` interface is still defined, but not referenced. And it now has the wrong return type of `bool` instead of `LSXSupportedOp` for the `Supported` function. Does it need to stick around, deleted, or updated?